multivac/sensors/test_status.py

```diff
@@ -58,15 +61,27 @@ def test_status_iter(log_fh, cache_filepath=None):
         cache = []
 
     hang_detected = False
+
+    saved_data = {}
     for line in log_fh:
         m = TEST_STATUS_LINE_RE.match(line)
+
         if m:
-            status = m.group('status') or 'fail'
+            if not m.group('status'):
+                saved_data.update({m['wid']: (m['test'], m['conf'])})
+                continue
+            status = m.group('status')
             res = (m['test'], m['conf'], status)
             if cache_filepath:
                 cache.append(res)
             yield res
             continue
+        elif saved_data:
+            status_match = LATE_STATUS.match(line)
+            if status_match:
+                get_test = saved_data.pop(status_match.group('wid'))
```

Comment:
`get_test` ([verb_..]) - good name for the function/method not for the variable

Reply:
fixed